src/lib/core.d.ts

```diff
@@ -1011,12 +1011,12 @@ interface Array<T> {
       * Combines two or more arrays.
       * @param items Additional items to add to the end of array1.
       */
-    concat<U extends T[]>(...items: U[]): T[];
+    concat<U extends T[]>(...items: (U | U[])[]): T[];

```

Comment:
Removing the generic line fixes that case, which makes me wonder whether the generic worked correctly anyways.

I'll ditch it in the PR because I don't know why it would be in there either.